commons/build.gradle

```diff
@@ -69,8 +69,13 @@ android {
     }
 
     composeOptions {
-        kotlinCompilerExtensionVersion '1.0.1'
+        kotlinCompilerExtensionVersion = "1.3.0"
     }
+
+    packagingOptions {
+        resources.excludes.add("META-INF/*")
+    }
```

Comment:
I think that we don't need packaging options here, because in buil time it is merged with builgradle from app